design/conop.md

```diff
@@ -40,4 +40,4 @@ Exhale valve moves to exhale position (fully open)
 Controlled adjusts pressure set point for closed loop control to PEEP pressure - controller adjusts blower speed to meet PEEP.
```

Comment:
The breath control cycles here are listed as run-on sentences. Separate into bullet points?